runtime/dart_isolate.h

```diff
@@ -28,6 +29,7 @@ namespace flutter {
 class DartVM;
 class DartIsolateGroupData;
 class IsolateConfiguration;
+enum class IsolateLaunchType;
```

Comment:
This shouldn't be forward declared here since it isn't referenced.

Reply:
Will remove, this was from an experiment.